docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md

```diff
@@ -0,0 +1,96 @@
+# How to update the coins file to work with the latest atomicDEX-API
+
+## Instructions
+
+- Compile the latest `mm2` binary following [these instructions](../atomicdex-setup/get-started-atomicdex.html)
+- Place the `coins` file in the old format in the same directory as the compiled `mm2` binary
+- Open a terminal and navigate to the directory with both the files.
+- Run the following command `./mm2 coins coins_new`
+- Output similar to the following will be shown. The "Success" message indicates that the conversion was successful
+
+```bash
+03 09:13:56, mm2:169] AtomicDEX MarketMaker f48b4ad DT 2020-07-28T17:25:18+07:00
+Success
+```
+
+- You will find a new file named `coins_new` in the same directory with the new format expected by AtomicDEX
```

Comment:
@gcharang Could you please add that user requires to replace `coins` with `coins_new` before attempting to run MM2?

Reply:
will do!